setup.cfg

```diff
@@ -1,9 +1,9 @@
 [metadata]
 name = rubrix
-# description = ...
-# long_description = ...
-# keywords = ...
-# license = ...
+description = Open-source tool for tracking, exploring and evolving data for AI projects.
```

Comment:
```suggestion
description = Open-source tool for tracking, exploring, and labeling data for AI projects.
```

Reply:
We can correct these typos en in Github's description too, if you want.